test/index.js

```diff
@@ -202,3 +217,34 @@ describe('proxy server rows', function () {
     await checkResponseCode(http, 'http://localhost:8008/solr/select?q=fhqwhagads', 200)
   })
 })
+
+describe('proxy server start', function () {
+  var proxy
+  var solrTestDouble
+
+  beforeEach(function () {
+    proxy = SolrProxy.start(null, { maxStart: 2000 })
+  })
+
+  afterEach(function () {
+    proxy.close()
+    if (solrTestDouble.close) {
+      solrTestDouble.close()
+    }
+  })
+
+  it('should return 403 if start param exceeds maximum', async function () {
+    solrTestDouble = createSolrTestDouble(200)
+    await checkResponseCode(http, 'http://localhost:8008/solr/select?q=fhqwhagads&start=2001', 403)
+  })
+
+  it('should return 200 if start param does not exceed maximum', async function () {
+    solrTestDouble = createSolrTestDouble(200)
+    await checkResponseCode(http, 'http://localhost:8008/solr/select?q=fhqwhagads&start=100', 200)
```

Comment:
Let's bump this up above the default but below the new value to make sure it is not just using the default of 1000.

```suggestion
    await checkResponseCode(http, 'http://localhost:8008/solr/select?q=fhqwhagads&start=1001', 200)
```